components/content-service/pkg/initializer/git.go

```diff
@@ -70,15 +70,11 @@ func (ws *GitInitializer) Run(ctx context.Context, mappings []archive.IDMapping)
 	}
 
 	gitClone := func() error {
-		if err := os.MkdirAll(ws.Location, 0775); err != nil {
+		if err := os.MkdirAll(ws.Location, 0755); err != nil {
```

Comment:
there was a bug when using 755 permission, that is why it was changed to 775

Reply:
Fixed